src/table/components/CellTextWrapper.tsx

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable react/require-default-props */
```

Comment:
This component is kind of temporary. I'm thinking this wrap/ellipsis solution should be used in all places, but can't do that yet for scrolling

Reply:
I think you are going to struggle to get this working perfectly with the virtual scroll table. There you need to calculate the width and height of a cell before it's rendered (Unless we somehow can get around that).

Given a certain width and text, you will need figure the number of lines the browser will use to render the text. Which might very well be vary between browsers as I don't think the all use the same word/line wrap algorithm.

My guess as to what will happen is that the browser might require 2-3 lines of text but our calculation might for example guess that it requires 1-2 lines.

But I think we should give it a try anyway!

Reply:
Well I mainly want to have the same css for word wrap/ellipsis, except that we need to break words in the scrolling solution. I'll leave this like this for now then and then see what I can do for the scrollign